kolibri/locale/es_419/LC_MESSAGES/django.po

```diff
@@ -157,8 +200,17 @@ msgid "You can also try updating your current browser."
 msgstr "También puedes actualizar tu navegador actual."
 
 #: kolibri/core/views.py:174
-msgid "No appropriate redirect pages found. It is likely that Kolibri is badly configured"
-msgstr "Fue imposible redirigir página correctamente. Es probable que Kolibri está mal configurado"
+msgid ""
+"No appropriate redirect pages found. It is likely that Kolibri is badly "
+"configured"
+msgstr ""
+"Fue imposible redirigir página correctamente. Es probable que Kolibri está "
+"mal configurado"
+
+#: kolibri/plugins/facility/views.py:38
+msgctxt "Name of a CSV file exporting the users of the facility"
+msgid "users_{}.csv"
+msgstr "usuarios_{}.csv"
 
 #: kolibri/plugins/oidc_provider_plugin/templates/oidc_provider/authorize.html:7
 msgid "OpenID Provider Authorization"
```

Comment:
Any reason `Provider Authorization` is capitalized? I suggest removing on both words.